lib/change-log.js

```diff
@@ -53,13 +59,13 @@ const _getRootEntityPathVals = function (txContext, entity, entityKey) {
 
 const _getAllPathVals = function (txContext) {
   const pathVals = []
-  const paths = txContext.path.split('/')
-  const entityIDs = _getEntityIDs(txContext.params)
+  const paths = [...txContext.path]
+  const entityIDs = [...txContext.params]
 
   for (let idx = 0; idx < paths.length; idx++) {
     const entity = getEntityByContextPath(paths.slice(0, idx + 1), txContext.hasComp)
     const entityID = entityIDs[idx]
-    const entityPathVal = `${entity.name}(${entityID})`
+    const entityPathVal = {target: entity.name, key: entityID};
```

Comment:
This illustrates one main change: paths are expressed as JSON arrays like 
```js
[{target: "RootEntity", key: {ID: "..."}}, {target: "Level1Entity", key: {ID: "..."}}]
```
instead of
```js
"RootEntity(...)/Level1Entity(...)"
```